src/test/suite/intraCellSetSelection.test.ts

```diff
@@ -24,7 +18,7 @@ async function runTest() {
 
   // FIXME: There seems to be some timing issue when you create a notebook
   // editor
-  await sleep(1000);
+  await sleepWithBackoff(1000);
```

Comment:
This will now cause it to sleep 250ms, then 500ms, 1s, 2s, 4s.  Hopefully that will both speed things up locally, and resolve the CI issues on Mac / Win